pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -453,7 +543,7 @@ func TestValidateThresholds(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		isValid := validateThresholds(test.input)
+		isValid := (validateThresholds(test.input) == nil)
 
 		if isValid != test.succeed {
```

Comment:
Please replace `succeed bool` with `error err` and set it to nil or a specific error instead given `validateThresholds` now returns `err`.

Reply:
Got it. I'll commit later.